src/cdi_ndi/NDI_Base.cc

```diff
@@ -9,11 +9,50 @@
 //----------------------------------------------------------------------------//
 
 #include "NDI_Base.hh"
+#include "ds++/DracoStrings.hh"
+#include "ds++/Query_Env.hh"
 #include "ds++/SystemCall.hh"
 #include "ds++/dbc.hh"
 
 namespace rtt_cdi_ndi {
 
+//----------------------------------------------------------------------------//
+// File local helper functions
+//----------------------------------------------------------------------------//
+
+#ifdef NDI_FOUND
+/*!
+ * \brief Warn if NDI library version doesn't match GENDIR version to 2 digits.
+ *
+ * \param[in] gendir path to the gendir file
+ * 
+ * This function is a no-op if \c NDI_FOUND is false. 
+ */
+void warn_ndi_version_mismatch(std::string const &gendir) {
+
+  std::string gendir_ver = rtt_dsxx::extract_version(gendir, 2);
```

Comment:
@KineticTheory Can `warn_ndi_version_mismatch` just be called in the `NDI_AtomicMass` constructor? It looks like duplicate code.

Reply:
Yeah.  I can make that change.  I didn't do it originally because I wanted this to be a file local function.  Give me a couple of minutes to fix this.

Reply:
Fixed.  Let's see what the CI reports.